lib/rust/ast/src/generation.rs

```diff
@@ -247,11 +247,12 @@ impl ScalaGenerator {
     /// Converts rust primitives into scala primitives i.e. `usize` => `Int`
     fn typ_name(&mut self, ident:&Ident) {
```

Comment:
You should note the limitations on types that can be converted in the documentation.